lib/core/src/Cardano/Wallet/Network.hs

```diff
@@ -176,6 +176,8 @@ data NetworkLayer m block = NetworkLayer
         :: TimeInterpreter (ExceptT PastHorizonException m)
     , syncProgress
         :: SlotNo -> m (SyncProgress)
+        -- ^ compute the ratio between the number in the argument and  
+        -- the total slots in the node
```

Comment:
FYI: https://github.com/input-output-hk/cardano-wallet/blob/60f5c7a04091163c1dea9d5a2bf50764129a51e6/lib/core/src/Cardano/Wallet/Primitive/SyncProgress.hs#L100-L111

Reply:
ah  ok, then I  will revert it, thanks

Reply:
Well, adding some comment is a good idea, but "the total slots in the node" is misleading and should be more "the slot corresponding to the wall-clock time".

Perhaps:

```suggestion
        -- ^ Compute the ratio between the provided 'SlotNo'  
        -- and the slot corresponding to the current wall-clock time.
        --
        -- See documentation in 'Cardano.Wallet.Primitive.SyncProgress.syncProgress' for more details.
```

And actually, there might be another relevant point which I made here in the implementation https://github.com/input-output-hk/cardano-wallet/blob/60f5c7a04091163c1dea9d5a2bf50764129a51e6/lib/shelley/src/Cardano/Wallet/Shelley/Network/Node.hs#L529-L536 but I guess should have made in the interface. So perhaps even:

```suggestion
        -- ^ Compute the ratio between the provided 'SlotNo'  
        -- and the slot corresponding to the current wall-clock time.
        --
        -- Unlike using 'Cardano.Wallet.Primitive.SyncProgress.syncProgress' after retrieving
        -- a 'timeInterpreter', this function will return 'NotResponding' rather than block in the edge
        -- case when the era history has not yet been fetched from the node on startup.
```